config/application.rb

```diff
@@ -19,5 +19,6 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
+    config.autoload_paths << Rails.root.join('lib')
```

Comment:
Does this mean the tests no longer have to include the `tei_helper` explicitly, because the classes in `lib` will be autoloaded?